### PR TITLE
a2fa 0.16.2 (new formula)

### PR DESCRIPTION
**Pull request by csyezheng**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/a/a2fa.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  def install
+    args = *std_go_args(ldflags: "-s -w")
+    args += ["-tags", "brew"] if OS.mac?
+    system "go", "build", *args, "./cmd/"
+  end
+
+  test do
+    expected = ""
+    output = shell_output("#{bin}/a2fa list")
+    assert_match expected, output
```
Comment (chenrui333) [inline Formula/a/a2fa.rb:20]:
```suggestion
    assert_empty shell_output("#{bin}/a2fa list")
```

**Comment by chenrui333** [comment]
@csyezheng thanks for the PR, can you amend the commit msg as `a2fa 0.16.2 (new formula)`

**Comment by csyezheng** [comment]
@chenrui333 Thanks for the suggestion, but my code repository doesn't seem to meet the requirements(30 forks, 30 watchers, 75 stars)

**Comment by chenrui333** [comment]
If you wish, you can host this formula in your own tap. Here are some docs to help you get started:

https://docs.brew.sh/Taps
https://docs.brew.sh/Interesting-Taps-and-Forks
https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap
https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/

**Comment by chenrui333** [comment]
Once it pass the notability check, happy to include into the core.

**Pull request closed without merging**